source/extensions/filters/http/jwt_authn/jwks_async_fetcher.cc

```diff
@@ -12,7 +12,13 @@ namespace JwtAuthn {
 namespace {
 
 // Default cache expiration time in 5 minutes.
-constexpr int PubkeyCacheExpirationSec = 600;
+constexpr int DefaultCacheExpirationSec = 600;
```

Comment:
style, for all constants:
```suggestion
constexpr std::chrono::seconds DefaultCacheExpiration{600};
```

Reply:
Done